tests/unit/core/TestScrollToTop.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```

Comment:
I'd expect `TestScrollToTop` to live at ` tests/unit/core/components/TestScrollToTop.js`